### ISSUE TITLE
Does imutils has a function like cv2.CAP_PROP_POS_FRAMES?

### ISSUE THREAD

**Issue by 666zcli** [jrosebr1/imutils#127]
Hi,
does the imutils has a function like cv2.CAP_PROP_POS_FRAMES to get a specific frame?
Thanks

**Comment by jrosebr1** [comment]
No, you would need to subclass or modify the `WebcamVideoStream` class to set any parameters of the `cv2.VideoCapture` object.

**Comment by 666zcli** [comment]
Thanks a lot
